types/cheesy-api/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+	"extends": "@definitelytyped/dtslint/dt.json",
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/cheesy-api/tslint.json;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.